src/foundry/client/data/documents/ambient-sound.d.mts

```diff
@@ -228,25 +228,23 @@ declare global {
       type OnDeleteOperation = Document.Database.OnDeleteOperation<Delete>;
     }
 
-    interface DatabaseOperations extends Document.Database.Operations<AmbientSoundDocument> {}
-
     /**
-     * @deprecated {@link ActorDelta.Types | `ActorDelta.SubType`}
+     * @deprecated - {@link AmbientSoundDocument.DatabaseOperation}
```

Comment:
No dashes after `@deprecated`. There's already one automatically added by the editor.